edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Docker/DockerEnvironment.cs

```diff
@@ -74,7 +74,7 @@ public async Task<ModuleSet> GetModulesAsync(CancellationToken token)
                     : moduleRuntimeInfo.ModuleStatus;
 
                 string image = !string.IsNullOrWhiteSpace(dockerRuntimeInfo.Config.Image) ? dockerRuntimeInfo.Config.Image : dockerModule.Config.Image;
-                var dockerReportedConfig = new DockerReportedConfig(image, dockerModule.Config.CreateOptions, dockerRuntimeInfo.Config.ImageHash);
+                var dockerReportedConfig = new DockerReportedConfig(image, dockerModule.Config.CreateOptions, dockerRuntimeInfo.Config.ImageHash, null);
```

Comment:
Why isn't dockerRuntimeInfo.NotaryContentTrust not used here? I think the EdgeAgent and EdgeHub specific tests in your PR would catch this if you passed in some value for them.